src/lib.rs

```diff
@@ -842,7 +842,8 @@ mod tests {
             #[test]
             fn $fn() {
                 crate::tests::panic_after(std::time::Duration::from_millis(500), || {
-                    match $crate::run_string($prog, true, Vec::new()) {
+                    let prog = std::concat!("q :: fn {", $prog, "\n{}\n}\nq()");
+                    match $crate::run_string(&prog, true, Vec::new()) {
```

Comment:
This needs to be fixed for real. But it might be a good way to atleast use some of the already written tests.